packages/api/src/beacon/routes/beacon/rewards.ts

```diff
@@ -54,18 +64,39 @@ export type Api = {
       HttpStatusCode.BAD_REQUEST | HttpStatusCode.NOT_FOUND
     >
   >;
+
+  /**
+   * Get sync committee rewards
+   * Returns participant reward value for each sync committee member at the given block. Optional filters can be
+   * passed in to filter the result
+   *
+   * @param blockId Block identifier.
+   * Can be one of: "head" (canonical head in node's view), "genesis", "finalized", \<slot\>, \<hex encoded blockRoot with 0x prefix\>.
+   * @param filters List of validator indices or pubkeys to filter in
+   */
+  getSyncCommitteeRewards(
+    blockId: BlockId,
+    filters?: ValidatorId[]
```

Comment:
> https://github.com/ChainSafe/lodestar/blob/2bb4ae6c9ee2fa464cca63084491b199143e6106/packages/api/src/beacon/routes/beacon/state.ts#L185-L189

`Indices` is definitely wrong, since they could be pubkeys too. And the parameter name in jsdoc doesn't match the actual parameter.

> https://github.com/ChainSafe/lodestar/blob/2bb4ae6c9ee2fa464cca63084491b199143e6106/packages/api/src/beacon/routes/beacon/state.ts#L39

I think `ids` should be used to emphasis it's an array of ids.

In this PR I will use `ids` for every usage of `ValidatorId[]`. The rest can be fixed in a separate PR

Reply:
> I think ids should be used to emphasis it's an array of ids.

I like it, it is most concise and an id (identifier) of a validator can either its pubkey or index

> In this PR I will use ids for every usage of ValidatorId[]. The rest can be fixed in a separate PR

Will address this is ssz refactor as there we pass args as an object and not separate parameters, so the naming becomes more important (hence I didn't like filters)